client/src/components/Offer/Offer.module.scss

```diff
@@ -0,0 +1,172 @@
+$isNewMarkColor: rgb(230, 73, 0);
```

Comment:
The name suggests that you've got a boolean value inside, which is not true. Something like new icon color would be also good.